swaglyrics/__main__.py

```diff
@@ -42,8 +48,13 @@ def main():
 		app.run(port=port)
 
 	elif args.cli:
-		song = spotify.song()  # get currently playing song
-		artist = spotify.artist()  # get currently playing artist
+
+		if args.song is None and args.artist is None:
```

Comment:
Once this is over the application keeps running and waiting for spotify to play a song instead of quitting

Reply:
```
		if args.song is None and args.artist is None:
			song = spotify.song()	# get currently playing song
			artist = spotify.artist()	# get currently playing artist
		else:
				song = args.song	# get song from command line argument
				artist = args.artist	# get artist from command line argument
				print(lyrics(song, artist))
				exit()
		print(lyrics(song, artist))
		print('\n(Press Ctrl+C to quit)')
```
This is closing the application after search result is shown.

Reply:
Right, I missed it. Sorry 😬

Reply:
Nope, you didn't miss anything, I just added this new part. 😂